src/components/datagrid/controls/display_selector.tsx

```diff
@@ -318,6 +360,22 @@ export const useDataGridDisplaySelector = (
             )}
           </EuiI18n>
         )}
+        {showResetButton && (
+          <EuiPopoverFooter>
+            <EuiFlexGroup justifyContent="flexEnd" responsive={false}>
+              <EuiFlexItem>
+                <EuiButtonEmpty
+                  flush="both"
+                  size="xs"
+                  onClick={resetToInitialState}
+                  data-test-subj="resetDisplaySelector"
+                >
+                  {resetButtonLabel}
+                </EuiButtonEmpty>
+              </EuiFlexItem>
+            <EuiFlexGroup>
```

Comment:
Looks like my code was missing a closing tag... 🤦‍♀️ 
```suggestion
            </EuiFlexGroup>
```